### DISCUSSION TITLE
5.8.6-1-MANJARO 无法启动

### DISCUSSION THREAD

**Discussion by wxfjamdc** [huan/docker-wechat#88] [Q&A]
🚀 Starting DoChat /dɑɑˈtʃæt/ ...

++ id -u
+ '[' 0 -ne 0 ']'
+ '[' -n 995 ']'
+ groupmod -o -g 995 audio
+ '[' -n 986 ']'
+ groupmod -o -g 986 video
++ id -g user
+ '[' 1001 '!=' 1000 ']'
+ groupmod -o -g 1001 group
++ id -u user
+ '[' 1000 '!=' 1000 ']'
+ chown user:group '/home/user/.wine/drive_c/users/user/Application Data' '/home/user/WeChat Files'
+ export HOSTNAME=DoChat
+ HOSTNAME=DoChat
+ echo DoChat
+ hostname DoChat
+ exec gosu user:group /entrypoint.sh
++ id -u
+ '[' 1000 -ne 0 ']'
+ '[' -n true ']'
+ wine reg query 'HKEY_CURRENT_USER\Software\Tencent\WeChat'
No protocol specified
reg��ϵͳ�޷��ҵ�ָ����ע�������ֵ
+ echo 'Register for Wechat not found ?'
+ exec wine 'C:\Program Files\Tencent\WeChat\WeChat.exe'
Register for Wechat not found ?
[1005/043316:FATAL:temp_window.cc(43)] Check failed: hwnd_. 
Backtrace:
	cef_time_to_timet [0x02D71391+635121]
	GetTXBrowsingService [0x02C84F2F+162895]
	GetTXBrowsingService [0x02C6BC31+59729]
	Func_initCef [0x067A1A80+336]
	_TlsGetData@12 [0x1078A31A+2989418]
	IMVQQEngine::`default constructor closure' [0x101FC0BF+1341599]
	IMVQQEngine::`default constructor closure' [0x101F9DCB+1332651]
	WCSGetInstance [0x107BDD9C+148460]
	WCSGetInstance [0x107BDCCB+148251]
	WCSGetInstance [0x107B9DE9+132153]
	WCSGetInstance [0x107BA39A+133610]
	IMVQQEngine::`default constructor closure' [0x10236988+1581416]
	IMVQQEngine::`default constructor closure' [0x101FC632+1342994]
	WCSGetInstance [0x107BF922+155506]
	(No symbol) [0x7F8BBB2C]
	(No symbol) [0x7F8BC239]
	(No symbol) [0x7F8BE553]
	(No symbol) [0x7F87C54E]
	(No symbol) [0x7F8832C9]
	(No symbol) [0x7F883537]
	(No symbol) [0x7F8B19F0]
	(No symbol) [0x7F8AAF39]
	WCSGetInstance [0x107BF510+154464]
	WCSGetInstance [0x107BA130+132992]
	_TlsGetData@12 [0x10795D2C+3037052]
	StartWachat [0x10796939+329]
	EntryPoint [0x00401918+18446744073709551615]
	EntryPoint [0x004030B9+18446744073709551615]
	(No symbol) [0x7B464A82]
	(No symbol) [0x7B466D5D]
	(No symbol) [0x7B464A8E]
	(No symbol) [0x00000000]

**Comment by walwan** [comment]
尝试安装x11-xserver-utils包，然后执行xhost +，再运行
